sdk/servicebus/service-bus/test/serviceBusClient.spec.ts

```diff
@@ -1,1140 +1,1133 @@
-        getReceiverClosedErrorMsg(
-          receiverClient.entityPath,
-          ClientType.QueueClient,
-          false,
-          TestMessage.sessionId
-        )
-      );
-      await testAllDispositions();
-    });
-  });
-
-  describe("Errors when creating second sender/receiver with first not closed", function(): void {
-    it("Open sender exists on QueueClient", async function(): Promise<void> {
-      await beforeEachTest(TestClientType.PartitionedQueue, TestClientType.PartitionedQueue, "");
-
-      await testCreateSender(
-        getOpenSenderErrorMsg(ClientType.QueueClient, senderClient.entityPath)
-      );
-    });
-
-    it("Open sender exists on TopicClient", async function(): Promise<void> {
-      await beforeEachTest(
-        TestClientType.PartitionedTopic,
-        TestClientType.PartitionedSubscription,
-        ""
-      );
-
-      await testCreateSender(getOpenSenderErrorMsg("TopicClient", senderClient.entityPath));
-    });
-
-    it("Open receiver exists on QueueClient #RunInBrowser", async function(): Promise<void> {
-      await beforeEachTest(TestClientType.PartitionedQueue, TestClientType.PartitionedQueue, "");
-
-      await testCreateReceiver(
-        getOpenReceiverErrorMsg(ClientType.QueueClient, receiverClient.entityPath)
-      );
-    });
-
-    it("Open receiver exists on SubscriptionClient", async function(): Promise<void> {
-      await beforeEachTest(
-        TestClientType.PartitionedTopic,
-        TestClientType.PartitionedSubscription,
-        ""
-      );
-
-      await testCreateReceiver(
-        getOpenReceiverErrorMsg(ClientType.SubscriptionClient, receiverClient.entityPath)
-      );
-    });
-
-    it("Open receiver exists for session on QueueClient #RunInBrowser", async function(): Promise<
-      void
-    > {
-      await beforeEachTest(
-        TestClientType.PartitionedQueueWithSessions,
-        TestClientType.PartitionedQueueWithSessions,
-        "",
-        true
-      );
-
-      await testCreateReceiver(
-        getOpenReceiverErrorMsg(
-          ClientType.QueueClient,
-          receiverClient.entityPath,
-          TestMessage.sessionId
-        ),
-        true
-      );
-    });
-
-    it("Open receiver exists for session on SubscriptionClient", async function(): Promise<void> {
-      await beforeEachTest(
-        TestClientType.PartitionedTopicWithSessions,
-        TestClientType.PartitionedSubscriptionWithSessions,
-        "",
-        true
-      );
-
-      await testCreateReceiver(
-        getOpenReceiverErrorMsg(
-          ClientType.SubscriptionClient,
-          receiverClient.entityPath,
-          TestMessage.sessionId
-        ),
-        true
-      );
-    });
-  });
-});
+// // Copyright (c) Microsoft Corporation. All rights reserved.
+// // Licensed under the MIT License.
+These tests are commented because the 2-client API version doesn't have the `ServiceBusClient` in it. We should re-viisit and update this once we have the top-level client.
```

Comment:
```suggestion
// These tests are commented because the 2-client API version doesn't have the `ServiceBusClient` in it. We should re-viisit and update this once we have the top-level client.
```